tools/deb/istio-iptables.sh

```diff
@@ -0,0 +1,87 @@
+#!/bin/bash
```

Comment:
Missing copyright. Please add copyrights wherever possible, esp since this is being supplied as a package.